app/models/orphan.rb

```diff
@@ -66,6 +66,8 @@ class Orphan < ActiveRecord::Base
   accepts_nested_attributes_for :current_address, allow_destroy: true
   accepts_nested_attributes_for :original_address, allow_destroy: true
 
+  default_scope { includes(:partner, :orphan_sponsorship_status, :orphan_status, original_address: :province) }
```

Comment:
To further clarify, I understand the concern, and had initially started with a narrower scope. The active admin filtering issue made this bubble up to the default scope.
Given this, I think the points in the post above hold true.